docs/utilities/parser.md

```diff
@@ -59,93 +47,16 @@ Use the decorator for fail fast scenarios where you want your Lambda function to
 ???+ note
     **This decorator will replace the `event` object with the parsed model if successful**. This means you might be careful when nesting other decorators that expect `event` to be a `dict`.
 
-```python hl_lines="18" title="Parsing and validating upon invocation with event_parser decorator"
-from aws_lambda_powertools.utilities.parser import event_parser, BaseModel
-from aws_lambda_powertools.utilities.typing import LambdaContext
-from typing import List, Optional
-
-import json
-
-class OrderItem(BaseModel):
-	id: int
-	quantity: int
-	description: str
-
-class Order(BaseModel):
-	id: int
-	description: str
-	items: List[OrderItem] # nesting models are supported
-	optional_field: Optional[str] # this field may or may not be available when parsing
-
-
-@event_parser(model=Order)
-def handler(event: Order, context: LambdaContext):
-	print(event.id)
-	print(event.description)
-	print(event.items)
-
-	order_items = [item for item in event.items]
-	...
-
-payload = {
-	"id": 10876546789,
-	"description": "My order",
-	"items": [
-		{
-			"id": 1015938732,
-			"quantity": 1,
-			"description": "item xpto"
-		}
-	]
-}
-
-handler(event=payload, context=LambdaContext())
-handler(event=json.dumps(payload), context=LambdaContext()) # also works if event is a JSON string
+```python hl_lines="21" title="Parsing and validating upon invocation with event_parser decorator"
+--8<-- "docs/examples/utilities/parser/parser_event_parser_decorator.py"
 ```
 
 ### parse function
 
 Use this standalone function when you want more control over the data validation process, for example returning a 400 error for malformed payloads.
 
-```python hl_lines="21 30" title="Using standalone parse function for more flexibility"
-from aws_lambda_powertools.utilities.parser import parse, BaseModel, ValidationError
-from typing import List, Optional
-
-class OrderItem(BaseModel):
-	id: int
-	quantity: int
-	description: str
-
-class Order(BaseModel):
-	id: int
-	description: str
-	items: List[OrderItem] # nesting models are supported
-	optional_field: Optional[str] # this field may or may not be available when parsing
-
-
-payload = {
-	"id": 10876546789,
-	"description": "My order",
-	"items": [
-		{
-			# this will cause a validation error
-			"id": [1015938732],
-			"quantity": 1,
-			"description": "item xpto"
-		}
-	]
-}
-
-def my_function():
-	try:
-		parsed_payload: Order = parse(event=payload, model=Order)
-		# payload dict is now parsed into our model
-		return parsed_payload.items
-	except ValidationError:
-		return {
-			"status_code": 400,
-			"message": "Invalid order"
-		}
+```python hl_lines="24 35" title="Using standalone parse function for more flexibility"
+--8<-- "docs/examples/utilities/parser/parser_parse_function.py"
```

Comment:
amazing PRs.
I think that you are missing in many examples the closing ``` at the end of the second line.
It doesnt display it nicely when omitted.

Reply:
Nope they are all their. Maybe the diff is not rendering right for you. It's easier to checkout the code and see

Reply:
dont see it.
here's another pr, and i see it there:
https://github.com/ran-isenberg/aws-lambda-handler-cookbook/pull/24/files

Reply:
@heitorlessa  can you check my eye sight lol?